crates/nu-cli/src/commands/math/eval.rs

```diff
@@ -91,7 +91,9 @@ pub async fn eval(
 }
 
 pub fn parse<T: Into<Tag>>(math_expression: &str, tag: T) -> Result<Value, String> {
-    match meval::eval_str(math_expression) {
+    let mut ctx = meval::Context::new();
+    ctx.var("tau", std::f64::consts::TAU);
```

Comment:
Looking at the error messages, it seems that this constant isn't available in stable Rust. I wonder if there are other ways to get to it (or if you could add it to the meval library.

Reply:
Doesn't this mean it's stable? https://doc.rust-lang.org/core/f64/consts/constant.TAU.html
I wonder if our CI isn't on 1.47 yet.

Reply:
It looks like 1.47 is installed on macos but i don't even see where rust is installed on linux or windows in the ci